cmd/spire-server/cli/agent/agent_test.go

```diff
@@ -0,0 +1,290 @@
+package agent
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/mitchellh/cli"
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/spire/cmd/spire-server/util"
+	common_cli "github.com/spiffe/spire/pkg/common/cli"
+	"github.com/spiffe/spire/pkg/server/api/agent/v1"
+	"github.com/spiffe/spire/pkg/server/api/rpccontext"
+	"github.com/spiffe/spire/pkg/server/plugin/datastore"
+	agentpb "github.com/spiffe/spire/proto/spire/api/server/agent/v1"
+	"github.com/spiffe/spire/proto/spire/common"
+	"github.com/spiffe/spire/test/clock"
+	"github.com/spiffe/spire/test/fakes/fakedatastore"
+	"github.com/spiffe/spire/test/fakes/fakeserverca"
+	"github.com/spiffe/spire/test/fakes/fakeservercatalog"
+	"github.com/spiffe/spire/test/spiretest"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+var (
+	td = spiffeid.RequireTrustDomainFromString("example.org")
+)
+
+type agentTest struct {
+	client  agentpb.AgentClient
+	ds      *fakedatastore.DataStore
+	testEnv *common_cli.Env
+
+	evictCmd,
+	listCmd,
+	showCmd cli.Command
+}
+
+func TestEvictHelp(t *testing.T) {
+	test := setupTest(t)
+
+	test.evictCmd.Help()
+	require.Equal(t, `Usage of agent evict:
+  -registrationUDSPath string
```

Comment:
may we keep this configuration as `registrationUDSPath`? right now it is no longer talking with registration api but to bundle api